### PR TITLE
[13.0][MIG] website_odoo_debranding

### PR DESCRIPTION
**Pull request by Tardo**
**Superseded** https://github.com/OCA/website/pull/676

cc @tecnativa TT21699

### CODE REVIEW

File: website_odoo_debranding/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -1,3 +1,3 @@
 This module remove Odoo branding on website:
 
-* Remove "Create a free website with Odoo" from footer
+* Remove "Powered by Odoo" from footer
```
Comment (chienandalu) [inline website_odoo_debranding/readme/DESCRIPTION.rst:3]: Missing space

**Review by sergio-teruel — APPROVED** [review]
Tested on runbot and code review

File: website_odoo_debranding/templates/disable_odoo.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <!-- Remove "Create a free website with Odoo" -->
+    <template
+        id="layout_footer_copyright"
+        inherit_id="website.brand_promotion"
+        customize_show="True"
```
Comment (chienandalu) [inline website_odoo_debranding/templates/disable_odoo.xml:7]: It isn't customizable indeed, isn't it?

Comment (pedrobaeza) [reply website_odoo_debranding/templates/disable_odoo.xml:7]: This is needed by multi-website and it's good to be able to disable this on tests. @Tardo have you tried about disabling this by default and enabling it on a hook only when no testing?

File: website_odoo_debranding/readme/DESCRIPTION.rst (hunk 1 above)
Comment (chienandalu) [reply website_odoo_debranding/readme/DESCRIPTION.rst:3]: Sorry. No here

File: website_odoo_debranding/readme/USAGE.rst [hunk 3]
```diff
@@ -1,3 +1 @@
-#. Go to your home page.
-#. Enable or disable *Customize > Footer Copyright > Remove Odoo Promotional
-   Link* at your will. It should be enabled by default.
+#.No additional configuration is needed. The legend "Powered by Odoo" from footer is removed automatically.
```
Comment (chienandalu) [inline website_odoo_debranding/readme/USAGE.rst]: Here's the missing space. Anyway, why this change of behavior?

File: website_odoo_debranding/templates/disable_odoo.xml (hunk 2 above)
Comment (chienandalu) [reply website_odoo_debranding/templates/disable_odoo.xml:7]: Ok. It wasn't working in my tests, and these readme lines seem to assert a change of behavior: https://github.com/OCA/website/pull/708/commits/8f6b731dd3e67fa419e9c5c9500c5bfcadee92b6#diff-9a72aa25a2cad81905ab4f260db75c13L2-L3

File: website_odoo_debranding/readme/USAGE.rst (hunk 3 above)
Comment (Tardo) [reply website_odoo_debranding/readme/USAGE.rst]: Now you can control the visibility in the "Customize" menu. Updated readme.

**Comment by Tardo** [comment]
Ok, updated to disable views in testing environments.

@chienandalu Load the page as public user, you will see this text disappear:
footer

**Review by manuelcalerosolis — APPROVED** [review]
